code/game/gamemodes/miniantags/hades/hades.dm

```diff
@@ -88,7 +125,12 @@
 			world << pick(creepyasssounds)
 		else
 			lastsinPerson = world.time
-			var/mob/living/carbon/human/sinPerson = pick(player_list)
+			var/mob/living/carbon/human/sinPerson = pick(living_mob_list)
+			var/depth = living_mob_list.len + 1 // just in case

```

Comment:
player_list contains ALL players, including lobby, ghosts etc. so including player_list is a baaad idea. It's what he had originally and it caused him to pick targets in the lobby and ghosts etc.